langium/src/extension/image-generation/image-generator.ts

```diff
@@ -85,7 +87,9 @@ export class ImageGenerator implements CommandUser, EventSubscriber<vscode.TextE
 		let log_level = this.configuration.getConfiguration(ConfigKey.LOGLEVEL);
 		
 
-		let command = 'java -jar "' + path.normalize(jar_file) + '" -i "' + path.normalize(input_file) + '" -d '+ diagram_name + ' --format ' + format + ' --log-level ' + log_level;
+
+		let command = java_version + ' -jar ' + jar_file + ' -i ' + input_file.path + ' -d '+ diagram_name + ' --format ' + format + ' --log-level ' + log_level;
+
```

Comment:
This would introduce a regression, it looks like there is a missing synchronization. Paths needs to be normalized to also work on Windows/Powershell platforms.

Reply:
This line actually does not compile, as input_file is now already a string and not an URI (since modif to support windows paths).
- paths must be normalized;
- options must be escaped with quotes  in case the user have a space in their paths.

It should be something like:

```
let command = java_version + ' -jar "' + path.normalize(jar_file) + '" -i "' + path.normalize(input_file) + '" -d '+ diagram_name + ' --format ' + format + ' --log-level ' + log_level;
```